tests/large_scale_evaluation.py

```diff
@@ -37,14 +37,14 @@
 from datetime import datetime
 from urllib.parse import urljoin
 
-import boto3
 import numpy as np
 import pandas as pd
+from scipy.stats import ks_2samp
+
```

Comment:
This resorting of the imports does not seem necessary. I suspect that the environment which you are using to test this does not have `boto3`, `botocore` and `tabulate` installed, which is why `isort` thinks they are first party dependencies. They should all be installed when you run `make install-develop`.